### DISCUSSION TITLE
How to map appdata or config folder of the JellyPlex-Watched to the host system?

### DISCUSSION THREAD

**Discussion by HHUBSS** [luigi311/JellyPlex-Watched#215] [General]
How can I map the appdata or config folder of the JellyPlex-Watched to the host system? Seems like it doesn't sync all watched content from Plex to Jellyfin and vice versa.

**Comment by luigi311** [comment]
Mapping the appdata/config folder shouldnt be necessary for iit to work as long as you set all the environment variables. If its not syncing for some reason you can try enabling the debuging and increasing the logging level just know it gets very verbose very quickly.
